doc/man1/openssl-kdf.pod.in

```diff
@@ -81,9 +81,8 @@ A key must be specified for most KDF algorithms.
 
 =item B<hexkey:>I<string>
 
-Specifies the secret key in hexadecimal form (two hex digits per byte).
-The key length must conform to any restrictions of the KDF algorithm.
-A key must be specified for most KDF algorithms.
+As before, but
```

Comment:
'As before' doesnt seem clearer to me..

Reply:
Similar to the I<key> option

Reply:
Good point, changed to `Similar to the B<key:> option`,
and likewise for the other three occurences.